x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/validate.test.ts

```diff
@@ -103,14 +103,29 @@ describe('validate', () => {
 
   describe('transformValidateFindAlerts', () => {
     test('it should do a validation correctly of a find alert', () => {
-      const findResult: FindResult = { data: [getResult()], page: 1, perPage: 0, total: 0 };
+      const findResult: FindResult = {
+        data: [getResult()],
+        page: 1,
+        perPage: 0,
+        total: 0,
+      };
       const [validated, errors] = transformValidateFindAlerts(findResult, []);
-      expect(validated).toEqual({ data: [ruleOutput], page: 1, perPage: 0, total: 0 });
+      expect(validated).toEqual({
+        data: [ruleOutput()],
+        page: 1,
+        perPage: 0,
+        total: 0,
+      });
```

Comment:
I'd type this so you have typescript safety like we have in the past as a `expected`:

```ts
const expected: FindResult = {
        data: [ruleOutput()],
        page: 1,
        perPage: 0,
        total: 0,
      }